system-variables.md

```diff
@@ -5429,6 +5429,25 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
     - `force_streaming`：TiFlash 直接将数据发送到下一阶段的 HashAgg，不进行预聚合操作
     - `auto`：TiFlash 根据当前工作负载的聚合度自动选择是否进行预聚合操作
 
+### `tiflash_hash_join_version` <span class="version-mark">从 v9.0.0 版本开始引入</span>
+
+> **警告：**
+>
+> 该变量控制的功能为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：是
+- 类型：枚举型
+- 默认值：`legacy`
+- 可选值：`legacy`、`optimized`
+- 控制 TiFlash 是否使用 Hash Join 算子的优化版。默认值为 `legacy`，代表不使用优化版。若设置为 `optimized`，TiFlash 在执行 Hash Join 算子时将使用其优化版，以提升 Hash Join 性能。
```

Comment:
```suggestion
- 控制 TiFlash 是否使用 [Hash Join 算子的优化版](/sql-statements/sql-statement-explain-analyze.md#hashjoinv2)。默认值为 `legacy`，代表不使用优化版。若设置为 `optimized`，TiFlash 在执行 Hash Join 算子时将使用其优化版，以提升 Hash Join 性能。
```

Reply:
`/sql-statements/sql-statement-explain-analyze.md#hashjoinv2` is about TiDB hash join v2, not TiFlash hash join v2.